### ISSUE TITLE
Select All for AntdSelect/AntdTreeSelect?

### ISSUE THREAD

**Issue by swt2c** [CNFeffery/feffery-antd-components#54]
I was wondering if you would consider adding a 'Select All' option to AntdSelect & AntdTreeSelect?  Something similar to:
https://codesandbox.io/s/ant-design-tree-select-select-all-dnekz?file=/index.js

It's not part of native Antd, obviously, so I can understand if you only want to wrap existing Antd Javascript APIs.  But I don't think it can be done from Python, unless I add another button or something.

Thanks for considering, and thanks a lot for your work!

**Comment by CNFeffery** [comment]
@swt2c it's easy to implement with the parameter `dropdownBefore` of `fac.AntdSelect`, here is a simple demo:

![demo](https://user-images.githubusercontent.com/49147660/218234871-b17e3096-9403-4c68-acff-5f634c71a1e7.gif)

```Python
import dash
from dash import html
import feffery_antd_components as fac
from dash.dependencies import Input, Output, State

app = dash.Dash(__name__)

app.layout = html.Div(
    [
        fac.AntdSelect(
            id='select-demo',
            options=[
                {
                    'label': f'option{i}',
                    'value': f'option{i}',
                }
                for i in range(1, 8)
            ],
            mode='multiple',
            maxTagCount='responsive',
            dropdownBefore=fac.AntdSpace(
                [
                    fac.AntdButton(
                        'Select all',
                        id='select-all',
                        type='link',
                        size='small'
                    ),
                    fac.AntdButton(
                        'Unselect all',
                        id='unselect-all',
                        type='link',
                        size='small'
                    ),
                    fac.AntdButton(
                        'Invert select',
                        id='invert-select',
                        type='link',
                        size='small'
                    )
                ],
                size='small',
                style={
                    'width': '100%'
                }
            ),
            style={
                'width': 265
            }
        )
    ],
    style={
        'padding': '100px'
    }
)


@app.callback(
    Output('select-demo', 'value'),
    [Input('select-all', 'nClicks'),
     Input('unselect-all', 'nClicks'),
     Input('invert-select', 'nClicks')],
    [State('select-demo', 'options'),
     State('select-demo', 'value')],
    prevent_initial_call=True
)
def handle_custom_select(*args):

    options, value = args[-2:]

    if dash.ctx.triggered_id == 'select-all':

        return [
            option['value']
            for option in options
        ]

    elif dash.ctx.triggered_id == 'unselect-all':

        return []

    elif dash.ctx.triggered_id == 'invert-select':

        return [
            option['value']
            for option in options
            if option['value'] not in (value or [])
        ]

    return dash.no_update


if __name__ == '__main__':
    app.run(debug=True)

```

**Comment by swt2c** [comment]
Thanks a lot for the hints!  I was going to ask if you could add `dropDownBefore` to `AntdTreeSelect` but it looks like you're already doing that.  :-)

**Comment by CNFeffery** [comment]
@swt2c Available in [0.2.0rc24](https://pypi.org/project/feffery-antd-components/0.2.0rc24/)
